examples/src/components/Tags.js

```diff
@@ -17,6 +18,20 @@ export default class Tags extends Component {
           options={{
             placeholder: 'search by tags',
           }} />
+          <br/>
+          <br/>
+          Using {'{'}id: 'x', text:'foo'{'}'}
+          <Select2
+            multiple={false}
+            data={[{text: 'Bug', id: 0}, {text: 'Feature', id: 1}, {text: 'Documents', id: 2}, {text: 'Discussion', id: 'UUID'}]}
```

Comment:
Please check the [js code style](https://github.com/rkit/react-select2-wrapper/blob/master/.eslintrc): Max length of line should be max 120